tests/Csv/LeagueCsvGeneratorTest.php

```diff
@@ -3,8 +3,9 @@
 namespace MarcusJaschen\Collmex\Tests\Csv;
 
 use MarcusJaschen\Collmex\Csv\LeagueCsvGenerator;
+use PHPUnit\Framework\TestCase;
 
-class LeagueCsvGeneratorTest extends \PHPUnit_Framework_TestCase
+class LeagueCsvGeneratorTest extends TestCase
```

Comment:
Doesn't this require an update of PHPUnit in the composer.json?

Reply:
Since the PHPUnit version is ```4.8.36```, the PHPUnit uses the new namespace.
You can see more details about the Travis build log.